python/ray/tune/result_grid.py

```diff
@@ -62,7 +62,6 @@ def _populate_exception(trial: Trial) -> Optional[TuneError]:
         return None
 
     def _trial_to_result(self, trial: Trial) -> Result:
-        # TODO(xwjiang): Use Kai's new checkpoint!
         result = Result(
             checkpoint=trial.checkpoint,
             metrics=trial.last_result,
```

Comment:
Is it possible to get back the entire sequence of reported metrics, not just the last result?